File: templates/template.rb
# run on ruby 2.0.0p645

def main()
  n = gets.to_i
  s = gets
  x = Array.new(n)
  y = Array.new(n)
  for i in 0...n do
    x[i], y[i] = gets.split.map(&:to_i)
  end
  # solve here
  puts n
  puts s
  puts x[0]
  puts y[n-1]
end

def solve(n)
	return 0
end

main()

File: templates/template.go
// run on go 1.11.4

package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

func next(scn *bufio.Scanner) string {
	scn.Scan()
	return scn.Text()
}

func nextInt(scn *bufio.Scanner) int {
	i, _ := strconv.Atoi(next(scn))
	return i
}

func main() {
	scn := bufio.NewScanner(os.Stdin)
	scn.Split(bufio.ScanWords)

	n := nextInt(scn)
	s := next(scn)
	x := make([]int, n)
	y := make([]int, n)
	for i := 0; i < n; i++ {
		x[i] = nextInt(scn)
		y[i] = nextInt(scn)
	}

	// solve here

	fmt.Println(n)
	fmt.Println(s)
	fmt.Println(x[0])
	fmt.Println(y[n-1])
}

func solve(n int) int {
	return 0
}
